test/PHPUnit/Controller/AbstractControllerTestCaseTest.php

```diff
@@ -618,6 +627,7 @@ public function testResetDoesNotCreateSessionIfNoSessionExists(): void
 
     /**
      * @psalm-return Generator<string, array{0: null|string}, mixed, void>
+     * @psalm-suppress PossiblyUnusedMethod Psalm does not yet recognize the DataProvider attribute
```

Comment:
It's cleaner to add this to `<issueHandlers>` with `<errorLevel type="suppress"><directory name="test" /></errorLevel>`, otherwise the code base gets littered with inline suppressions. At some point, the PHPUnit plugin will get updated and someone will have to remove each inline suppression.

Reply:
okay, it's your call. I'd suppress such things directly in the code and when it's updated, `findUnusedBaselineEntry` will complain anyways. Suppressing in the `psalm.xml` file is not as granular, it will suppress even valid `PossiblyUnusedMethod` errors; and when it's fixed, the `UnusedIssueHandlerSuppression` error is harder to read, because it just says `psalm.xml:0:0` 🙂

Reply:
It's less future maintenance work and Psalm not finding dead code in `/test` is an acceptable tradeoff I think.

Reply:
okay!